system/config/Config.cfc

```diff
@@ -1324,7 +1324,7 @@ component {
 		derivatives.openGraphImage = {
 			  permissions = "inherit"
 			, autoQueue   = [ "image" ]
-			, transformations = [ { method="shrinkToFit", args={ width=400, height=400 } } ]
+			, transformations = [ { method="shrinkToFit", args={ width=1200, height=630 } } ]
```

Comment:
Ideally this is dealt with in a separate ticket. That can be communicated. Also, based on the guidelines, `shrinkToFit 1200x1200` seems more appropriate to work with different orientations? The dimensions wouldn't always be exactly what is called for, but should be good enough.